dist/war/getstarted/index.html

```diff
@@ -1,90 +1,156 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="">
-    <meta name="author" content="ProActive">
-    <link rel="shortcut icon" href="assets/image/favicon.ico">
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="description" content="">
+  <meta name="author" content="Activeeon">
+  <link rel="shortcut icon" href="assets/image/favicon.ico">
 
-    <title>Get started with ProActive</title>
+  <title>Get started with ProActive</title>
 
-    <link href="assets/css/bootstrap.css" rel="stylesheet">
-    <link href="assets/css/custom.css" rel="stylesheet">
+  <link rel="stylesheet" href="assets/css/bootstrap4.1.3//bootstrap.min.css">
```

Comment:
you did not add this file in your pull request.

You need to to git add -f assets/css/bootstrap4.1.3//bootstrap.min.css I think

Reply:
Required resources are added correctly now

Reply:
@nebil, watch out for `//bootstrap.min` !, remove the extra slash